pkg/cstor/pvc_operation.go

```diff
@@ -162,6 +165,15 @@ func (p *Plugin) createPVC(volumeID, snapName string) (*Volume, error) {
 	if err = p.waitForAllCVR(vol); err != nil {
 		return nil, err
 	}
+
```

Comment:
This is a multi-step procedure. Needs a closer look to see if this function (CreatePVC) is idempotent. For example what happens after the PVC is created and velero pod crashes. In the background the PVC and required PVs may get created, but Velero plugin on restart - will again start the process of CreatePVC, and as there is a a PVC that exists, might fail the restore.

Reply:
right, This check needs to be executed if PVC exist.